plasmapy/formulary/radiation.py

```diff
@@ -0,0 +1,160 @@
+    .. math::
+        E_1 (y) = - \int_{-y}^\infty \frac{e^{-t}}{t}dt
+
+    and y is the dimensionless argument
+
+    .. math::
+        y = \frac{1}{2} \frac{\omega^2 m_e}{k_{max}^2 k_B T_e}
+
+    where   :math:`k_{max}` is a maximum wavenumber approximated here as
+    :math:`k_{max} = 1/\lambda_B` where  :math:`\lambda_B` is the electron
+    de Broglie wavelength.
+
+    Parameters
+    ----------
+
+    frequencies : `~astropy.units.Quantity`
+        Array of frequencies over which the bremsstrahlung spectrum will be
+        calculated (convertable to Hz).
+
+    n_e : `~astropy.units.Quantity`
+        Electron number density in the plasma (convertable to m^-3)
+
+    T_e : `~astropy.units.Quantity`
+        Temperature of the electrons (in K or convertible to eV)
+
+    n_i : `~astropy.units.Quantity` (optional)
+        Ion number density in the plasma (convertable to m^-3). Defaults
+        to the quasi-neutral conditon n_i=n_e/Z.
+
+    ion : str or `~plasmapy.particles.Particle`, optional
+        An instance of `~plasmapy.particles.Particle`, or a string
+        convertible to `~plasmapy.particles.Particle`.
+
+    kmax :  `~astropy.units.Quantity`
+        Cutoff wavenumber (convertable to u.rad/u.m). Defaults to the inverse
+        of the electron de Broglie wavelength.
+
+    Returns
+    -------
+
+    spectrum : `~astropy.units.Quantity`
+        Computed bremsstrahlung spectrum over the frequencies provided.
+
+    Notes
+    -----
+
+    For details, see "Radiation Processes in Plasmas" by
+    Bekefi. `ISBN 978\\-0471063506`_.
+
+    .. _`ISBN 978\\-0471063506`: https://ui.adsabs.harvard.edu/abs/1966rpp..book.....B/abstract
+    """
+
+    # Default n_i is n_e/Z:
+    if n_i is None:
+        n_i = n_e / ion_species.integer_charge
+
+    # Default value of kmax is the electrom thermal de Broglie wavelength
+    if kmax is None:
+        kmax = (np.sqrt(const.m_e.si * const.k_B.si * T_e) / const.hbar.si).to(1 / u.m)
+
+    # Convert frequencies to angular frequencies
+    ω = (frequencies * 2 * np.pi * u.rad).to(u.rad / u.s)
+
+    # Calculate the electron plasma frequency
+    ω_pe = plasma_frequency(n=n_e, particle="e-")
+
+    # Check that all ω < wpe (this formula is only valid in this limit)
+    if np.min(ω) < ω_pe:
+        raise PhysicsError(
+            "Lowest frequency must be larger than the electron "
+            f"plasma frequency {ω_pe:.1e}, but min(ω) = {np.min(ω):.1e}"
+        )
+
+    # Check that the parameters given fall within the Rayleigh-Jeans limit
+    # hω << kT_e
+    rj_const = (
+        np.max(ω) * const.hbar.si / (2 * np.pi * u.rad * const.k_B.si * T_e)
+    ).to(u.dimensionless_unscaled)
+    if rj_const.value > 0.1:
+
+        raise PhysicsError(
+            "Rayleigh-Jeans limit not satisfied: "
+            "hbar*ω/kT_e = {rj_const.value:.2e} > 0.1. "
+            "Try lower ω or higher T_e."
+        )
+
+    # Calculate the bremsstralung power spectral density in several steps
```

Comment:
```suggestion
    # Calculate the bremsstrahlung power spectral density in several steps
```
Ha, finally putting those hours on duolingo to good use!

Reply:
ARGH! I came so close to not misspelling this in the entire PR!